src/language-js/options.js

```diff
@@ -76,16 +76,17 @@ module.exports = {
     type: "choice",
     default: [
       { since: "0.0.0", value: false },
-      { since: "0.19.0", value: "none" }
+      { since: "0.19.0", value: "none" },
+      { since: "2.0.0", value: "es5" }
     ],
     description: "Print trailing commas wherever possible when multi-line.",
     choices: [
-      { value: "none", description: "No trailing commas." },
       {
         value: "es5",
         description:
           "Trailing commas where valid in ES5 (objects, arrays, etc.)"
       },
+      { value: "none", description: "No trailing commas." },
```

Comment:
Why is this moved?

Reply:
playground shows none above es5

Reply:
I’d&nbsp;prefer if&nbsp;the&nbsp;order wasn’t changed, as&nbsp;that&nbsp;way, `es5`&nbsp;is&nbsp;still implied to&nbsp;be&nbsp;between `none`&nbsp;and&nbsp;`all`.

Reply:
@ExE-Boss why? after changing default to `es5`, shouldn't we put it on the top?